system/Helpers/url_helper.php

```diff
@@ -540,3 +540,39 @@ function url_title($str, $separator = '-', $lowercase = FALSE)
 }
 
 //--------------------------------------------------------------------
+if ( ! function_exists('_stringify_attributes'))
+{
+	/**
+	 * Stringify attributes for use in HTML tags.
+	 *
+	 * Helper function used to convert a string, array, or object
+	 * of attributes to a string.
+	 *
+	 * @param	mixed	string, array, object
+	 * @param	bool
+	 * @return	string
+	 */
+	function _stringify_attributes($attributes, $js = FALSE)
```

Comment:
I don't know about you, but I'd get rid of the leading underscore. We don't use that style anywhere else in the current codebase.